src/ops/inc_multihead_self_attention.cu

```diff
@@ -1366,28 +1354,15 @@ void peft_bwd_kernel(IncMultiHeadSelfAttentionMeta const *m,
       int m_ = m->qSize;
       int n_ = num_tokens;
       int k_ = m->num_q_heads * (m->qProjSize + m->kProjSize + m->vProjSize);
-      int lda = m_;
-      int ldb = n_;
-      int ldc = m_;
-      checkCUDA(cublasGemmEx(m->handle.blas,
-                             CUBLAS_OP_N,
-                             CUBLAS_OP_T,
-                             m_,
-                             n_,
-                             k_,
-                             &alpha,
-                             A,
-                             cublas_data_type,
-                             lda,
-                             B,
-                             cublas_data_type,
-                             ldb,
-                             &beta,
-                             C,
-                             cublas_data_type,
-                             ldc,
-                             compute_type,
-                             CUBLAS_GEMM_DEFAULT_TENSOR_OP));
+
+      // TODO: checkout if the input grad ptr has some relation with
```

Comment:
can you replace these comments with an explanation of what this transposeAdd is doing and why it is needed?